global.json

```diff
@@ -1,6 +1,6 @@
 {
-  "projects": [ "src" ],
-    "sdk": {
+  "projects": [ "src", "tests" ],
```

Comment:
@JimBobSquarePants does this change break something? Do you know why "tests" was not present in the original list?